test/profileCache.js

```diff
@@ -64,12 +64,13 @@ function makeProfileReq(uid) {
 
 describe('profile cache', function() {
   beforeEach(() => {
-    Server = require('./lib/server');
     clearRequireCache();
+    Server = require('./lib/server');
   });
 
   afterEach(() => {
     mock.done();
+    require('../lib/db')._teardown();
```

Comment:
This was a head-scratcher - the new test runner would hang when executing these tests, but only if they ran *after* some other tests.  I eventually realized doing `require('./lib/server')` and then `clearRequireCache`, was causing it to import a copy of `./lib/db` and then discard the reference to it.  We thus couldn't close its connection pool and would cause the tests to hang.  It's fixed by clearing the require cache before loading `Server`, and by tearing down the db module after each test rather than only once at the end.

I'm not thrilled about the way a bunch of implied shared state leaks between tests like this, but fixing that seemed like a bigger refactor than I was willing to take on...